ccl/oidcccl: TestOIDCEnabled failed

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/12136711?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/12136711?buildTab=artifacts#/) on release-23.1 @ [30cd7fc605e1d05f4a37750806052625d7b9fae1](https://github.com/cockroachdb/cockroach/commits/30cd7fc605e1d05f4a37750806052625d7b9fae1):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/c73670e949b6ee7ec5feb14dc370d013/logTestOIDCEnabled3719791519
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/c73670e949b6ee7ec5feb14dc370d013/logTestOIDCEnabled3719791519
--- FAIL: TestOIDCEnabled (5.77s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.00s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/unowned
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-32256

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/12137664?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/12137664?buildTab=artifacts#/) on release-23.1 @ [30cd7fc605e1d05f4a37750806052625d7b9fae1](https://github.com/cockroachdb/cockroach/commits/30cd7fc605e1d05f4a37750806052625d7b9fae1):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/c73670e949b6ee7ec5feb14dc370d013/logTestOIDCEnabled2240289487
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/c73670e949b6ee7ec5feb14dc370d013/logTestOIDCEnabled2240289487
--- FAIL: TestOIDCEnabled (7.04s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.00s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12137982?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12137982?buildTab=artifacts#/) on release-23.1 @ [30cd7fc605e1d05f4a37750806052625d7b9fae1](https://github.com/cockroachdb/cockroach/commits/30cd7fc605e1d05f4a37750806052625d7b9fae1):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled3187331360
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled3187331360
--- FAIL: TestOIDCEnabled (13.02s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.02s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12137983?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12137983?buildTab=artifacts#/) on release-23.1 @ [30cd7fc605e1d05f4a37750806052625d7b9fae1](https://github.com/cockroachdb/cockroach/commits/30cd7fc605e1d05f4a37750806052625d7b9fae1):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled198949609
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled198949609
--- FAIL: TestOIDCEnabled (12.60s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.02s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/12148408?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/12148408?buildTab=artifacts#/) on release-23.1 @ [1d5f07d043035d14735c328a59384cd96cb47b09](https://github.com/cockroachdb/cockroach/commits/1d5f07d043035d14735c328a59384cd96cb47b09):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/c73670e949b6ee7ec5feb14dc370d013/logTestOIDCEnabled3866565858
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/c73670e949b6ee7ec5feb14dc370d013/logTestOIDCEnabled3866565858
I231012 01:31:28.352421 9865 ccl/oidcccl/claim_match.go:78  [-] 2  extracted SQL username myfakeemail from the target claim email
E231012 01:31:28.352493 9866 ccl/oidcccl/claim_match.go:87  [-] 3  OIDC: failed to complete authentication: expected one group in regexp
I231012 01:31:28.352580 9867 ccl/oidcccl/claim_match.go:78  [-] 4  extracted SQL username myfakeemail from the target claim groups
E231012 01:31:28.352626 9868 ccl/oidcccl/claim_match.go:87  [-] 5  OIDC: failed to complete authentication: expected one group in regexp
--- FAIL: TestOIDCEnabled (9.44s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.01s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_UnitTests_BazelUnitTests/12149346?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_UnitTests_BazelUnitTests/12149346?buildTab=artifacts#/) on release-23.1 @ [1d5f07d043035d14735c328a59384cd96cb47b09](https://github.com/cockroachdb/cockroach/commits/1d5f07d043035d14735c328a59384cd96cb47b09):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled406486363
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled406486363
--- FAIL: TestOIDCEnabled (8.03s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.00s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12149685?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12149685?buildTab=artifacts#/) on release-23.1 @ [1d5f07d043035d14735c328a59384cd96cb47b09](https://github.com/cockroachdb/cockroach/commits/1d5f07d043035d14735c328a59384cd96cb47b09):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled2459341018
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled2459341018
--- FAIL: TestOIDCEnabled (5.80s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.00s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12149686?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12149686?buildTab=artifacts#/) on release-23.1 @ [1d5f07d043035d14735c328a59384cd96cb47b09](https://github.com/cockroachdb/cockroach/commits/1d5f07d043035d14735c328a59384cd96cb47b09):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled473588934
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled473588934
--- FAIL: TestOIDCEnabled (12.89s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.00s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12149687?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12149687?buildTab=artifacts#/) on release-23.1 @ [1d5f07d043035d14735c328a59384cd96cb47b09](https://github.com/cockroachdb/cockroach/commits/1d5f07d043035d14735c328a59384cd96cb47b09):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled1046168781
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled1046168781
--- FAIL: TestOIDCEnabled (13.26s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.01s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12162959?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12162959?buildTab=artifacts#/) on release-23.1 @ [1d5f07d043035d14735c328a59384cd96cb47b09](https://github.com/cockroachdb/cockroach/commits/1d5f07d043035d14735c328a59384cd96cb47b09):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled1949687247
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled1949687247
--- FAIL: TestOIDCEnabled (14.59s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.00s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12162958?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12162958?buildTab=artifacts#/) on release-23.1 @ [1d5f07d043035d14735c328a59384cd96cb47b09](https://github.com/cockroachdb/cockroach/commits/1d5f07d043035d14735c328a59384cd96cb47b09):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled359646358
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled359646358
--- FAIL: TestOIDCEnabled (6.01s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.00s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12162960?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12162960?buildTab=artifacts#/) on release-23.1 @ [1d5f07d043035d14735c328a59384cd96cb47b09](https://github.com/cockroachdb/cockroach/commits/1d5f07d043035d14735c328a59384cd96cb47b09):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled3147007239
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled3147007239
--- FAIL: TestOIDCEnabled (12.69s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.02s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/12166912?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/12166912?buildTab=artifacts#/) on release-23.1 @ [72e5b81dcf5672237cb50dde5309b3d1796bdf08](https://github.com/cockroachdb/cockroach/commits/72e5b81dcf5672237cb50dde5309b3d1796bdf08):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/c73670e949b6ee7ec5feb14dc370d013/logTestOIDCEnabled3777689801
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/c73670e949b6ee7ec5feb14dc370d013/logTestOIDCEnabled3777689801
--- FAIL: TestOIDCEnabled (7.14s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.00s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/12172330?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/12172330?buildTab=artifacts#/) on release-23.1 @ [5fca9ee5fa80d35338ae34b7ff123587b9c3aaee](https://github.com/cockroachdb/cockroach/commits/5fca9ee5fa80d35338ae34b7ff123587b9c3aaee):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/c73670e949b6ee7ec5feb14dc370d013/logTestOIDCEnabled3961574819
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/c73670e949b6ee7ec5feb14dc370d013/logTestOIDCEnabled3961574819
--- FAIL: TestOIDCEnabled (6.42s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.00s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12177588?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12177588?buildTab=artifacts#/) on release-23.1 @ [366fe013b4b63bef98c953d03b302dc5e0b13ee7](https://github.com/cockroachdb/cockroach/commits/366fe013b4b63bef98c953d03b302dc5e0b13ee7):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled3404700540
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled3404700540
--- FAIL: TestOIDCEnabled (13.84s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.00s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12177587?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12177587?buildTab=artifacts#/) on release-23.1 @ [366fe013b4b63bef98c953d03b302dc5e0b13ee7](https://github.com/cockroachdb/cockroach/commits/366fe013b4b63bef98c953d03b302dc5e0b13ee7):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled3162343740
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled3162343740
--- FAIL: TestOIDCEnabled (5.95s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.00s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12177589?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12177589?buildTab=artifacts#/) on release-23.1 @ [366fe013b4b63bef98c953d03b302dc5e0b13ee7](https://github.com/cockroachdb/cockroach/commits/366fe013b4b63bef98c953d03b302dc5e0b13ee7):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled1846028865
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled1846028865
--- FAIL: TestOIDCEnabled (13.12s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.01s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12186246?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12186246?buildTab=artifacts#/) on release-23.1 @ [366fe013b4b63bef98c953d03b302dc5e0b13ee7](https://github.com/cockroachdb/cockroach/commits/366fe013b4b63bef98c953d03b302dc5e0b13ee7):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled500847994
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled500847994
--- FAIL: TestOIDCEnabled (5.67s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.00s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12186247?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12186247?buildTab=artifacts#/) on release-23.1 @ [366fe013b4b63bef98c953d03b302dc5e0b13ee7](https://github.com/cockroachdb/cockroach/commits/366fe013b4b63bef98c953d03b302dc5e0b13ee7):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled621516020
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled621516020
--- FAIL: TestOIDCEnabled (12.16s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.00s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12186248?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12186248?buildTab=artifacts#/) on release-23.1 @ [366fe013b4b63bef98c953d03b302dc5e0b13ee7](https://github.com/cockroachdb/cockroach/commits/366fe013b4b63bef98c953d03b302dc5e0b13ee7):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled3009415687
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled3009415687
--- FAIL: TestOIDCEnabled (11.61s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.01s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12196109?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12196109?buildTab=artifacts#/) on release-23.1 @ [366fe013b4b63bef98c953d03b302dc5e0b13ee7](https://github.com/cockroachdb/cockroach/commits/366fe013b4b63bef98c953d03b302dc5e0b13ee7):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled3532517465
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled3532517465
--- FAIL: TestOIDCEnabled (5.66s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.00s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12196110?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12196110?buildTab=artifacts#/) on release-23.1 @ [366fe013b4b63bef98c953d03b302dc5e0b13ee7](https://github.com/cockroachdb/cockroach/commits/366fe013b4b63bef98c953d03b302dc5e0b13ee7):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled715374893
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled715374893
--- FAIL: TestOIDCEnabled (13.54s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.01s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12196111?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12196111?buildTab=artifacts#/) on release-23.1 @ [366fe013b4b63bef98c953d03b302dc5e0b13ee7](https://github.com/cockroachdb/cockroach/commits/366fe013b4b63bef98c953d03b302dc5e0b13ee7):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled868126900
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/3ed8b395931fde05baf66bd2e4d88d97/logTestOIDCEnabled868126900
--- FAIL: TestOIDCEnabled (12.09s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.02s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/12200343?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/12200343?buildTab=artifacts#/) on release-23.1 @ [426a545d22fb1332b76cb51c4130bad3d1038dc2](https://github.com/cockroachdb/cockroach/commits/426a545d22fb1332b76cb51c4130bad3d1038dc2):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/c73670e949b6ee7ec5feb14dc370d013/logTestOIDCEnabled3486601531
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/c73670e949b6ee7ec5feb14dc370d013/logTestOIDCEnabled3486601531
--- FAIL: TestOIDCEnabled (6.50s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.00s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/12202762?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/12202762?buildTab=artifacts#/) on release-23.1 @ [e2c7b6e0629f25c933f25f37a19c5ca592f0babd](https://github.com/cockroachdb/cockroach/commits/e2c7b6e0629f25c933f25f37a19c5ca592f0babd):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/c73670e949b6ee7ec5feb14dc370d013/logTestOIDCEnabled1354583871
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/c73670e949b6ee7ec5feb14dc370d013/logTestOIDCEnabled1354583871
--- FAIL: TestOIDCEnabled (6.89s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.00s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/oidcccl.TestOIDCEnabled [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/12203443?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/12203443?buildTab=artifacts#/) on release-23.1 @ [73eaab1f25f2bb68399feaa67868095c93a33494](https://github.com/cockroachdb/cockroach/commits/73eaab1f25f2bb68399feaa67868095c93a33494):


```
=== RUN   TestOIDCEnabled
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/c73670e949b6ee7ec5feb14dc370d013/logTestOIDCEnabled457916973
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
=== CONT  TestOIDCEnabled
    authentication_oidc_test.go:244: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/c73670e949b6ee7ec5feb14dc370d013/logTestOIDCEnabled457916973
I231016 16:35:08.692670 7284 ccl/oidcccl/claim_match.go:78  [-] 2  extracted SQL username myfakeemail from the target claim email
--- FAIL: TestOIDCEnabled (6.75s)
=== RUN   TestOIDCEnabled/login_redirect
    authentication_oidc_test.go:238: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/oidcccl/authentication_oidc_test.go:238
        	Error:      	Not equal: 
        	            	expected: "some/random/path"
        	            	actual  : "/"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-some/random/path
        	            	+/
        	Test:       	TestOIDCEnabled/login_redirect
    --- FAIL: TestOIDCEnabled/login_redirect (0.00s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestOIDCEnabled.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/oidcccl/authentication_oidc_test.go
+++ b/pkg/ccl/oidcccl/authentication_oidc_test.go
@@ -88,7 +88,11 @@ func TestOIDCEnabled(t *testing.T) {
 	defer log.Scope(t).Close(t)
 
 	ctx := context.Background()
-	s, db, _ := serverutils.StartServer(t, base.TestServerArgs{})
+	s, db, _ := serverutils.StartServer(t, base.TestServerArgs{
+		// OIDC does not yet support multi-tenancy with independent
+		// configurations. Only system tenant OIDC is currently supported.
+		DisableDefaultTestTenant: true,
+	})
 	defer s.Stopper().Stop(ctx)
 
 	newRPCContext := func(cfg *base.Config) *rpc.Context {

